netdata.spec.in

```diff
@@ -147,25 +147,25 @@ Requires(pre): /usr/sbin/useradd
 # to support weak dependencies. Explicitly requiring our default plugins
 # makes it impossible to properly test the packages prior to upload,
 # so we just skip depending on them on CentOS 7.
-%if 0%{?centos_ver} != 7
 %if 0%{?_have_ebpf}
-Recommends: netdata-plugin-ebpf
+Requires: netdata-plugin-ebpf
 %endif
-Recommends: netdata-plugin-apps
-Recommends: netdata-plugin-pythond
-Recommends: netdata-plugin-go
-Recommends: netdata-plugin-debugfs
+Requires: netdata-plugin-apps
+Requires: netdata-plugin-pythond
+Requires: netdata-plugin-go
+Requires: netdata-plugin-debugfs
 %if 0%{?_have_freeipmi}
-Suggests:   netdata-plugin-freeipmi
+Requires:   netdata-plugin-freeipmi
```

Comment:
Should `netdata-plugin-freeipmi` stay _Suggests_? It is _Suggests_ on Debian.

There is a problem with installing `netdata-plugin-freeipmi` by default - it uses (tested on centos7, deb11 VMs) 10% of a single core for 1 minute on every (re)start (after it it exits because of failed data collection).